.github/workflows/publish-docker-image.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Check out the repo
         uses: actions/checkout@v2
       - name: Build and push to GitHub Container Registry
-        uses: docker/build-push-action@v1
+        uses: docker/build-push-action@v2.4.0
```

Comment:
v1 is in fact better than v2 for this particular GitHub Action. Should add an ignore file for Dependabot

Reply:
@dependabot ignore this major version